notes.md

```diff
@@ -0,0 +1,12 @@
+TODO:
+
+* decide on synthdef submission method,
+  (eg. sent by clients? or already loaded on server?)
+* double check the SynthDescRelay
+
+Notable Changes:
+
+1/8/18 james:
```

Comment:
This type of information is great for a commit message. In my experience, putting it in a separate file becomes hard to maintain.